packages/nekoton_bridge/bin/merger.dart

```diff
@@ -0,0 +1,200 @@
+import 'dart:io';
+
+const mergedFileName = 'merged';
+final importsReg = RegExp('.*use (?:.*?|\n)*?;');
+
+/// This runs automatically in the beginning of build.rs script
+/// [Directory.current] must be /nekoton_bridge/native
+void main() async {
+  final nativeSrcDir = Directory('${Directory.current.path}/src');
+  final mergedFile = File('${nativeSrcDir.path}/$mergedFileName.rs');
+  final mergedFileSink = mergedFile.openWrite();
+
+  final contentBuffer = StringBuffer();
+
+  /// Collection of imported local crates in all *api.rs files.
+  /// Key - name of global crate like crate or nekoton_wrapper or utils or nekoton
+  /// Value - hierarchy of sub-modules
+  final crates = <String, ModuleHierarchy>{};
+
+  for (final entry in nativeSrcDir.listSync(recursive: true)) {
+    if (entry.path.endsWith('api.rs') && entry is File) {
+      final content = entry.readAsStringSync();
+
+      /// Collect structure of imports
+      importsReg.allMatches(content).forEach(
+            (importMatch) => parseSingleImport(
+              content.substring(importMatch.start, importMatch.end),
+              crates,
+            ),
+          );
+
+      /// Only code that should be saved to result file
+      final pureContent = content
+          // all imports
+          .replaceAll(importsReg, '')
+          // all mods
+          .replaceAll(RegExp('.*mod.*;'), '')
```

Comment:
Maybe do it consistently by taking out strings to constants, like mergedFileName?